vars/runATH.groovy

```diff
@@ -12,7 +12,7 @@ def call(Map params = [:]) {
     def jdks = params.get('jdks', [8])
     def athContainerImageTag = params.get("athImage", "jenkins/ath");
     def configFile = params.get("configFile", null)
-    def defaultJavaOptions = params.get('javaOptions', ['--no-transfer-progress'])
+    def defaultJavaOptions = params.get('javaOptions', [])
```

Comment:
https://github.com/timja/pipeline-library/blob/03ff5064bc29f9cf283d6a7d516be706f04ff546/vars/runATH.groovy#L219 sets JVA options - there should be a `MAVEN_OPTIONS` argument?

Reply:
could be 😄 
but better to add for everyone I think if we want it, i.e. https://github.com/jenkinsci/acceptance-test-harness/pull/710